<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargeDialog</class>
 <widget class="QDialog" name="ChargeDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>563</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit Charges</string>
  </property>
  <widget class="QPushButton" name="chgNewButt">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>520</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chgSaveButt">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>520</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chgDelButt">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>520</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="Q3ButtonGroup" name="chgCustType">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>240</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Customer Type</string>
   </property>
   <property name="exclusive">
    <bool>true</bool>
   </property>
   <widget class="QRadioButton" name="chgSellButt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Seller</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="chgBuyrButt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Buyer</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="moveChargeDownButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>\/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveChargeUpButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>520</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>/\</string>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>230</width>
     <height>230</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QCheckBox" name="chgReSale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Charge on Re-Sale</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chgNoSale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Charge on No-Sale</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chgPairs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Double on Pairs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chgMult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiplier</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chgComm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add To Commission</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isBeefCouncil">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Beef Council</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="textLabel3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>280</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Default Expression:
(0 = manual; else H,W,T,*,+,-,/)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="chgEditAmt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>280</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Expressions in the auction program&lt;br&gt;are calculations that you wish to&lt;br&gt;be performed automatically. In this&lt;br&gt;case the variable are:&lt;BR&gt;H =&gt; head, W =&gt; weight,&lt;BR&gt;T =&gt; transaction total</string>
   </property>
  </widget>
  <widget class="Q3Table" name="breakTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>280</width>
     <height>111</height>
    </rect>
   </property>
   <property name="numRows">
    <number>11</number>
   </property>
   <property name="numCols">
    <number>2</number>
   </property>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Expression</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Break</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="chgEditName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="Q3Table" name="codeTable">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>260</width>
     <height>260</height>
    </rect>
   </property>
   <property name="numRows">
    <number>10</number>
   </property>
   <property name="numCols">
    <number>10</number>
   </property>
   <row>
    <property name="text">
     <string>0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>0</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>5</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>6</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>7</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>8</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>9</string>
    </property>
   </column>
  </widget>
  <widget class="Q3ListBox" name="chgListBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>500</width>
     <height>190</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="codeRelLineEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>180</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3_2_2_3">
   <property name="geometry">
    <rect>
     <x>554</x>
     <y>180</y>
     <width>146</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Quick Codes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="colorComboBox">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>140</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0-69</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70-79</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80-89</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90-99</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Code</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>220</y>
     <width>260</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sex</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel3_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>30</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string>F
l
a
w</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="breakTypeCombo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Head</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Avg Weight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Total</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="textLabel1_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>510</y>
     <width>320</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;small&gt;&lt;B&gt;&lt;p color=&quot;0xff0000&quot;&gt;WARNING: Changing the order of the charges mid-sale &lt;u&gt;WILL&lt;/u&gt; cause undesired side effects.&lt;/p&gt;&lt;/b&gt;&lt;/small&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>160</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Break At:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_4_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Scope:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="scopeCombo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Transaction</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Invoice</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chgEditName</tabstop>
  <tabstop>chgEditAmt</tabstop>
  <tabstop>chgMult</tabstop>
  <tabstop>chgNoSale</tabstop>
  <tabstop>chgReSale</tabstop>
  <tabstop>chgPairs</tabstop>
  <tabstop>codeRelLineEdit</tabstop>
  <tabstop>colorComboBox</tabstop>
  <tabstop>chgListBox</tabstop>
  <tabstop>codeTable</tabstop>
  <tabstop>chgSellButt</tabstop>
  <tabstop>chgNewButt</tabstop>
  <tabstop>chgDelButt</tabstop>
  <tabstop>chgSaveButt</tabstop>
 </tabstops>
 <includes>
  <include location="local">dao/chargedao.h</include>
  <include location="global">list</include>
  <include location="global">string</include>
  <include location="local">types/animalcode.h</include>
  <include location="global">vector</include>
 </includes>
 <resources/>
 <connections/>
</ui>
